### PR TITLE
Fix getNodes of RangeSelection

### PR DESCRIPTION
**Pull request by kosei28**
This fixes a problem in RangeSelection's getNodes method, which incorrectly retrieves the nodes when the anchor is after the focus.

### CODE REVIEW

**Comment by acywatson** [comment]
Thanks for the PR - can you make an issue or a failing test to demonstrate the problem? It's easier to review the change with some context around the issue you're trying to address.

**Comment by kosei28** [comment]
@acywatson 
I made the issue. #3973

File: packages/lexical/src/LexicalSelection.ts [hunk 1]
```diff
@@ -600,25 +600,31 @@ export class RangeSelection implements BaseSelection {
...
     const anchor = this.anchor;
     const focus = this.focus;
+    const isBefore = anchor.isBefore(focus);
     let firstNode = anchor.getNode();
     let lastNode = focus.getNode();
+    let firstOffset = anchor.offset;
+    let lastOffset = focus.offset;
+
+    if (!isBefore) {
+      [firstNode, lastNode] = [lastNode, firstNode];
```
Comment (acywatson) [inline packages/lexical/src/LexicalSelection.ts:610]: nit: the typical pattern we use here is ternary assignment conditioned on isBefore. I think it's a bit clearer and consistent with what we do elsewhere.

**Comment by thegreatercurve** [comment]
LGTM. Will merge if @acywatson is happy as well?

**Pull request merged**